source/manual/load-test.html.md

```diff
@@ -0,0 +1,116 @@
+---
+owner_slack: "#govuk-developers"
+title: Run a load test with k6
+section: Testing
+layout: manual_layout
+parent: "/manual.html"
+---
+
+These instructions are based on [this blog post](https://grafana.com/blog/2022/06/23/running-distributed-load-tests-on-kubernetes/).
+
+1. Log into AWS and set the context to the environment you wish to test:
+
+    ```sh
+    ENVIRONMENT=staging
+    ROLE=admin
+    AWS_REGION=eu-west-1
```

Comment:
**question:** variable usage

Is the `AWS_REGION` environment variable used under the hood somewhere? I can't see where it's used here

Reply:
I don't believe it is.  I'll remove this as the config was taken from my notes for doing some other things in AWS.